src/manifest/index.ts

```diff
@@ -18,7 +18,7 @@ const findManifestDependencies = async ({ manifest, isProduction, isStrict, dir,
 
   const scripts = Object.entries(manifest.scripts ?? {}).reduce((scripts, [scriptName, script]) => {
     if (script && (scriptFilter.length === 0 || scriptFilter.includes(scriptName))) {
-      return [...scripts, script];
+      scripts.push(script);
     }
     return scripts;
   }, [] as string[]);
```

Comment:
FWIW, this is more or less what `flatMap`'s made for

```js
  const scripts = Object.entries(manifest.scripts ?? {}).flatMap(([scriptName, script]) => {
    if (script && (scriptFilter.length === 0 || scriptFilter.includes(scriptName))) {
      return [script];
    }

    return [];
  });
```

Reply:
I think this code should be slower, because we still create new array.